plugins/media/wowza/services/LiveConversionProfileService.php

```diff
@@ -177,9 +177,10 @@ public function serveAction($streamName, $hostname = null)
 
 	protected function appendLiveParams(LiveStreamEntry $entry, WowzaMediaServerNode $mediaServer = null, SimpleXMLElement $encodes, liveParams $liveParams)
 	{
+		$conversionExtraParam = json_decode($liveParams->getConversionEnginesExtraParams());
 		$streamName = $entry->getId() . '_' . $liveParams->getId();
 		$videoCodec = 'PassThru';
-		$audioCodec = (strpos($liveParams->getConversionEnginesExtraParams(), 'audioPassthrough') !== false) ? 'PassThru' : 'AAC';
+        $audioCodec = ($conversionExtraParam !== null && $conversionExtraParam->audioPassthrough !== null) ? 'PassThru' : 'AAC';
```

Comment:
To avoid writing long if you can simply do
($conversionExtraParam && $conversionExtraParam->audioPassthrough)